packages/perseus/src/widgets/interactive-graphs/graphs/components/__snapshots__/movable-line.test.tsx.snap

```diff
@@ -330,7 +330,7 @@ exports[`Rendering Does render extensions of line when option is enabled 1`] = `
         />
         <g
           class="interactive-graph-arrowhead"
-          transform="translate(0 0) rotate(-45)"
+          transform="translate(0 0) rotate(0)"
```

Comment:
The line associated with these arrowheads are drawn with weird point references ([0,0], [0,0]) ([see preceding `<line>` element](https://github.com/Khan/perseus/blob/f67858a4c7bdd76116aa79eebbad298227f0a86b/packages/perseus/src/widgets/interactive-graphs/graphs/components/__snapshots__/movable-line.test.tsx.snap#L325-L330)). This line has no slope, so the rotation should be zero.